data/resources/ui/window.blp

```diff
@@ -66,95 +83,129 @@ template GeopardWindow: Adw.ApplicationWindow {
           margin-end: 6;
           spacing: 6;
           styles ["headerbar"]
+
           Gtk.WindowControls ws {
             side: start;
             visible: bind ws.empty inverted;
           }
+
           [title]
           Gtk.SearchEntry small_url_bar {
             hexpand: true;
+            placeholder-text: _("Enter Gemini address or search with Geminispace");
           }
+
           Gtk.WindowControls we {
             side: end;
             visible: bind we.empty inverted;
           }
         }
       }
     }
+
     Adw.TabBar tab_bar {
       view: tab_view;
     }
+
     Gtk.Overlay {
       Adw.TabView tab_view {
         vexpand: true;
         hexpand: true;
       }
+
       [overlay]
       Gtk.ProgressBar progress_bar {
         styles ["osd"]
         valign: start;
         text: bind GeopardWindow.progress;
       }
+
       [overlay]
       Gtk.Box {
         orientation: vertical;
         valign: end;
-        Gtk.Box url_status_box {
-          visible: false;
-          styles ["background"]
-          Gtk.Label url_status {
-            xalign: 0.0;
-            wrap: true;
-            margin-top: 6;
-            margin-bottom: 6;
-            margin-start: 6;
-            margin-end: 6;
+
+        Gtk.Box {
+          orientation: horizontal;
+
+          Gtk.Box url_status_box {
+            visible: false;
+            styles ["osd"]
```

Comment:
I'm not sure I want this. I know most browsers use a style similar to the "osd" class you applied here, but I think having a black rectangle popping up every time you hover a link ruins the experience a bit.

Reply:
Hmm, I could try to slightly darken website color on-the-fly in a different variable and set it instead of `osd` class, so it won't be as distracting, but it still will be visible as a seperate object from webview.